index.html

```diff
@@ -0,0 +1,291 @@
+          <span class="line"></span>
+          <div class="sectionfour__container__partners">
+            <div class="sectionfour__container__partners__partner">
+              <div
+                class="sectionfour__container__partners__partner__responsive"
+              >
+                <img
+                  src="static/imgs/company-1.png"
+                  alt="company-1"
+                  class="responsive sectionfour__container__partners__partner__responsive__respo"
+                />
+              </div>
+            </div>
+            <div class="sectionfour__container__partners__partner">
+              <div
+                class="sectionfour__container__partners__partner__responsive"
+              >
+                <img
+                  src="static/imgs/company-2.png"
+                  alt="company-2"
+                  class="responsive sectionfour__container__partners__partner__responsive__respo"
+                />
+              </div>
+            </div>
+            <div class="sectionfour__container__partners__partner">
+              <div
+                class="sectionfour__container__partners__partner__responsive"
+              >
+                <img
+                  src="static/imgs/company-3.png"
+                  alt="company-3"
+                  class="responsive sectionfour__container__partners__partner__responsive__respo"
+                />
+              </div>
+            </div>
+            <div class="sectionfour__container__partners__partner">
+              <div
+                class="sectionfour__container__partners__partner__responsive"
+              >
+                <img
+                  src="static/imgs/company-4.png"
+                  alt="company-4"
+                  class="responsive sectionfour__container__partners__partner__responsive__respo"
+                />
+              </div>
+            </div>
+            <div class="sectionfour__container__partners__partner">
+              <div
+                class="sectionfour__container__partners__partner__responsive"
+              >
+                <img
+                  src="static/imgs/company-5.png"
+                  alt="company-5"
+                  class="responsive sectionfour__container__partners__partner__responsive__respo"
+                />
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- end sectionfour -->
+    </main>
+    <!-- end main -->
+    <!-- start footer -->
+    <footer class="footer" id="footer">
+      <div class="container">
+        <div class="footer__container__parfoot">
+          <div class="footer__container__parfoot__foot">
+            <div class="footer__container__parfoot__foot__responsive">
+              <img
+                src="static/imgs/company-1.png"
+                alt=""
+                class="responsive footer__container__parfoot__foot__responsive__respo"
+              />
+            </div>
```

Comment:
- [ ] I have realized that this image is not visible. Kindly change it  to ensure it is visible

**This is your design:**

![image](https://user-images.githubusercontent.com/86318284/197014802-f1225e4b-9d2e-465d-9276-d0b638469443.png)

**This is the desired design**

![image](https://user-images.githubusercontent.com/86318284/197014949-cc296179-600c-43dc-b8e4-7c0738866b0d.png)

**Note** this does not mean that you use exact same images. It merely means use an image that contrasts well with the background to be visible